packages/web/pages/assets/[denom].tsx

```diff
@@ -421,35 +420,51 @@ const TokenChartSection = () => {
 const TokenChartHeader = observer(() => {
   const { assetInfoConfig } = useAssetInfoView();
 
-  const minimumDecimals = 2;
-  const maxDecimals = Math.max(
-    getDecimalCount(
-      (assetInfoConfig.hoverPrice?.toDec() ?? new Dec(0)).toString()
-    ),
-    minimumDecimals
+  const { data, isLoading } = api.edge.assets.getAssetPrice.useQuery(
```

Comment:
NIT: may be a bit more readable if we rename `data` to `data: assetPrice`

Reply:
Done